### PR TITLE
Docs: adding multi-root editor to predefined builds. 

### LINKED ISSUE #13536: Add multi root editor description to predefined builds section
## 📝 Provide a description of requested docs changes

The multi-root editor is missing from the [predefined builds](https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/predefined-builds.html) guide and should be added. This was extracted as a follow-up issue from [the main issue](https://github.com/ckeditor/ckeditor5/issues/11493).

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: adding multi-root editor to predefined builds. Closes #13536.

---

### Additional information

_Only proceed after_ #13532 _is merged._

### CODE REVIEW

**Review by scofalik — CHANGES_REQUESTED** [review]
One note regarding the image. IIRC, in the example that I prepared, the bottom boxes shoud have the same height.

File: docs/installation/getting-started/predefined-builds.md [hunk 1]
```diff
@@ -466,6 +467,78 @@ Full code example:
...
+In your HTML page add an element that CKEditor 5 should make editable:
+
+```html
+<div id="editor"></div>
+```
+
+Load the multi-root editor build (here, the [CDN](https://cdn.ckeditor.com/) location is used):
+
+```html
+<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/multi-root/ckeditor.js"></script>
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/installation/getting-started/predefined-builds.md:492]: Ask Robert about it. I believe.

File: docs/installation/getting-started/predefined-builds.md [hunk 2]
```diff
@@ -503,6 +577,7 @@ The table below presents the list of all plugins included in various builds.
 				<td style="text-align:center; width:70px">❌</td>
 				<td style="text-align:center; width:70px">❌</td>
 				<td style="text-align:center; width:70px">✅</td>
+				<td style="text-align:center; width:70px">❌</td>
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/installation/getting-started/predefined-builds.md:580]: I checked it.

File: docs/installation/getting-started/predefined-builds.md [hunk 3]
```diff
@@ -466,6 +467,78 @@ Full code example:
...
+### Multi-root editor
+
+Multi-root editor is an editor type that features multiple, separate editable areas.
+
+The main difference between using a multi-root editor and using multiple separate editors is the fact that in a multi-root editor all editable areas belong to the same editor instance share the same configuration, toolbar and the undo stack, and produce one document.
+
+
+{@img assets/img/editor-multi-root.png 925 Screenshot of a multi-root editor.}
+
+To try it out online, check the {@link examples/builds/inline-editor inline editor example}.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/installation/getting-started/predefined-builds.md]: In commit.

File: docs/installation/getting-started/predefined-builds.md [hunk 4]
```diff
@@ -466,6 +467,78 @@ Full code example:
...
+
+#### Installation example
+
+In your HTML page add an element that CKEditor 5 should make editable:
+
+```html
+<div id="editor"></div>
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/installation/getting-started/predefined-builds.md:485]: In commit.

File: docs/installation/getting-started/predefined-builds.md [hunk 5]
```diff
@@ -466,6 +467,78 @@ Full code example:
...
+
+Alternatively, you may install CKEditor 5 from `npm`:
+
+```bash
+npm install --save @ckeditor/ckeditor5-build-multi-root
+```
+
+Then bundle it together with your app.
+
+Call the {@link module:editor-inline/inlineeditor~InlineEditor#create `InlineEditor.create()`} method.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/installation/getting-started/predefined-builds.md]: In commit.

File: docs/installation/getting-started/predefined-builds.md [hunk 6]
```diff
@@ -466,6 +467,78 @@ Full code example:
...
+
+```html
+<script>
+	InlineEditor
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/installation/getting-started/predefined-builds.md]: In commit.

File: packages/ckeditor5-editor-multi-root/README.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ CKEditor 5 multi-root editor implementation
 [![Build Status](https://travis-ci.com/ckeditor/ckeditor5.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5)
 ![Dependency Status](https://img.shields.io/librariesio/release/npm/@ckeditor/ckeditor5-editor-multi-root)
 
-The multi-root editor implementation for CKEditor 5.
+The multi-root editor build for CKEditor 5. Read more about the [multi-root editor build](https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/predefined-builds.html#multi-root-editor) and see the [demo](https://ckeditor.com/docs/ckeditor5/latest/examples/builds/multi-root-editor.html).
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-editor-multi-root/README.md]: In commit.

**Comment by godai78** [comment]
The recent push concerns cksource/ckeditor5-internal#2797

**Pull request merged**